### PR TITLE
ui: Copy deposit address alert

### PR DESCRIPTION
**Pull request by martonp**
Adds a disappearing message confirming that the deposit address has been copied.

### CODE REVIEW

**Comment by ukane-philemon** [comment]
Nice, I was actually looking out for this :).

File: client/webserver/site/src/js/wallets.ts [hunk 1]
```diff
@@ -233,8 +233,13 @@ export default class WalletsPage extends BasePage {
     Doc.hide(this.page.forms)
   }
 
-  copyAddress () {
-    navigator.clipboard.writeText(this.page.depositAddress.textContent || '')
+  async copyAddress () {
+    const page = this.page
+    navigator.clipboard.writeText(page.depositAddress.textContent || '')
+    Doc.show(page.copyAlert)
```
Comment (chappjc) [inline client/webserver/site/src/js/wallets.ts:239]: We should only show the "Copied!" alert if `writeText` says it succeeded.

```js
navigator.clipboard.writeText(textContent).then(() => {
	Doc.show(page.copyAlert)
	...
}, (reason) => {
	console.error('Unable to copy:', reason) // or a different alert, whatever
})
```

https://github.com/decred/dcrdata/commit/c9521242ea01902ed3452250306d67ac54932824

File: client/webserver/site/src/css/wallets.scss [hunk 2]
```diff
@@ -92,3 +92,9 @@ table.wallets {
   cursor: pointer;
   color: #a8a8a8;
 }
+
+#copyAlert {
+  margin-left: 9px;
```
Comment (chappjc) [inline client/webserver/site/src/css/wallets.scss:97]: With 9x it's right up against the `div` for me.  Maybe 12?

**Review by chappjc — APPROVED** [review]
Working for me, but the alter text is a little close to the address text div.  A couple more pixels of margin and I think it would be a little better.

https://user-images.githubusercontent.com/9373513/175057130-35bae9f6-7bbd-4752-a9dc-d7bb42c73d9a.mp4

File: client/webserver/site/src/css/wallets.scss (hunk 2 above)
Comment (martonp) [reply client/webserver/site/src/css/wallets.scss]: 12 looks good.

**Pull request merged**